graphstorm-processing/graphstorm_processing/config/numerical_configs.py

```diff
@@ -92,3 +92,53 @@ def __init__(self, config: Mapping):
         self.separator = self._transformation_kwargs.get("separator", None)
 
         self._sanity_check()
+
+
+class BucketFeatureConfig(FeatureConfig):
```

Comment:
Let's call this `BucketNumericalFeatureConfig` to be consistent with naming elsewhere.